src/main/resources/api_configuration.json

```diff
@@ -35,6 +35,60 @@
       "method": "GET",
       "pathPattern": "/orders/validate",
       "proxyPath": "/ebsconet/validate"
+    },
+    {
+      "type": "MOSAIC",
```

Comment:
These endpoints are generic and can be applied for any order integration, not only for Mosaic, can we name type like "GENERIC" or "COMMON" in this case?